### PR TITLE
Delete sandbox cluster

### PR DESCRIPTION
**Pull request by ale-novo**
this pr is to delete any reference to the sandbox cluster, witch was deleted.

issue: https://github.com/ministryofjustice/cloud-platform/issues/113

### CODE REVIEW

File: cluster-components/kuberos/ingress.yml [hunk 1]
```diff
@@ -1,15 +0,0 @@
-apiVersion: extensions/v1beta1
```
Comment (alkar) [inline cluster-components/kuberos/ingress.yml]: It should be safe to remove all the files under `cluster-components/kuberos/`

File: concourse/Concoursefile.yaml [hunk 2]
```diff
@@ -1,51 +0,0 @@
-resource_types:
```
[the remark this answers is not in the document]
Comment (ale-novo) [reply concourse/Concoursefile.yaml]: github_concourse.ini points to the sandbox cluster, and according to raz this was an investigation and its not being used

File: cluster-components/kuberos/ingress.yml (hunk 1 above)
Comment (ale-novo) [reply cluster-components/kuberos/ingress.yml]: ok

Comment (ale-novo) [reply cluster-components/kuberos/ingress.yml]: fixed

**Pull request merged**